apiv6/generate_server.sh

```diff
@@ -15,6 +15,7 @@ function run() {
     generate_cloud_export_server
     generate_synthetics_server
     go fmt ./...
+    git apply ./localhost_apiserver/patches/0001-Modify-localhost_apiserver-not-to-require-preset-in-.patch
```

Comment:
Why is application of this patch required, when you also actually changed the: apiv6/localhost_apiserver/synthetics/go/api_synthetics_admin_service.go

Does it make any change it that way?

Reply:
This way, every time the schema changes, the file will be regenerated accordingly. After regeneration is done, the patch with the fix is applied. I chose this way in order to keep the file up to date with the latest schema.

Other solution would be to disable generation of this file by adding it to `.openapi-generator-ignore`.